PostHog / Restack announcement (#3415)
PostHog/posthog.com: contents/blog/posthog-restack-announcement.md

---

> date: 2022-05-09

Reviewer: ```suggestion
date: 2022-05-11
```

title: Restack joins the PostHog Marketplace
rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
author: ["joe-martin"]
featuredImage: ../images/blog/posthog-engineering-blog.png
featuredImageType: full
categories: ["Product updates"]
---

> Over the last few months, we’ve steadily grown the number of businesses offering services through [the PostHog Marketplace](/marketplace). We’ve added partnerships with teams such as [Altinity](/blog/posthog-altinity-announce), Birch, TenRec and, as of today, [Restack](/marketplace/restack). 

Reviewer: ```suggestion
Over the last few months, we’ve steadily grown the number of businesses offering services through [the PostHog Marketplace](/marketplace). We’ve added partnerships with teams such as [Altinity](/blog/posthog-altinity-announce), Birch, TenRec and, as of today, [Restack](/marketplace/restack). 
```

Isn't Tenrec still 'coming soon'?

Author: ```suggestion
Over the last few months, we’ve steadily grown the number of businesses offering services through [the PostHog Marketplace](/marketplace). We’ve added partnerships with teams such as [Altinity](/blog/posthog-altinity-announce), Birch and, as of today, [Restack](/marketplace/restack). 
```

I mean, it's still listed, but good point. Have removed.

[Restack](https://www.restack.io/) is a platform for deploying open-source software stacks simply and effectively on Kubernetes, and is run by a team of open-source enthusiasts. Through the PostHog Marketplace, Restack enables you to easily setup, run and maintain a PostHog instance on your own Kubernetes Cluster on AWS. 

> In short, Restack enables you to deploy and benefit from a self-hosted PostHog instance, but without the maintenance busy work. Best of all, Restack Console ensures that deployments are [HIPAA](/docs/privacy/hipaa-compliance), FISMA and SOC2 compliant — handy if you deal with a lot of [PII](/blog/what-is-personal-data-pii).

Author: ```suggestion
In short, Restack enables you to deploy and benefit from a self-hosted PostHog instance, but without the maintenance busy work. Best of all, Restack Console can ensure that deployments are [HIPAA](/docs/privacy/hipaa-compliance), FISMA and SOC2 compliant — handy if you deal with a lot of [PII](/blog/what-is-personal-data-pii).
```
